### PR TITLE
Slele/create model

### PR DESCRIPTION
**Pull request by JeetRH**
Data and code for the hands on lab to create a Pytorch model

### CODE REVIEW

**Review by rsriniva — APPROVED** [review]
looks good to me.  I'll ask Jaime to test this hands on lab

**Comment by rsriniva** [comment]
@jramcast - please also test this PR for creating models using Pytorch

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Exploration\n",
+    "\n",
+    "In this notebook, we'll take the opportunity to explore the data we will be using.  Data exploration and understanding is a fundamental step in Machine Learning.\n",
+    "\n",
+    "In this learning path, the data we'll be looking at is the Diabetes data set.  It consists of ~800 rows of medical data (e.g. blood pressure, heart rate etc...) for female patients with and without diabetes.  It is worth mentioning that in ML, 800 examples is considered a very very small dataset.  We'll be creating a model that will help us determine which of our patients, based on their medical data, may have diabetes.\n",
```
Comment (jramcast) [inline 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb:11]: We might want to provide a reference to the dataset (a url or a cite to the authors)

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#plot histograms of the columns on multiple subplots\n",
+    "plt.close('all')\n",
+    "df.diff().hist(color=\"k\", alpha=0.5, bins=20)"
```
Comment (jramcast) [inline 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb]: let's make this histogram a bit bigger to prevent label overlap

image

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "corrM = df.corr()\n",
+    "corrM"
```
Comment (jramcast) [inline 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb]: We can improve the output a bit by adding some style to the correlation matrix 
image

File: 3.create-model/pytorch-lab/notebooks/02-model-development.ipynb [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+    "\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import accuracy_score"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we have imported our required lbraries and packages we load the data into a dataframe (df)."
```
Comment (jramcast) [inline 3.create-model/pytorch-lab/notebooks/02-model-development.ipynb]:
```suggestion
    "After we have imported our required libraries and packages we load the data into a dataframe (df)."
```

**Review by jramcast — APPROVED** [review]
These notebooks are great. Simple and to the point.

Left some minor comments. No need for rereview.

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb (hunk 1 above)
Comment (JeetRH) [reply 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb:11]: Added a reference for the dataset

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb (hunk 2 above)
Comment (JeetRH) [reply 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb]: Added a fig_size to this which makes these histograms look really nice and with no overlaps

File: 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb (hunk 3 above)
Comment (JeetRH) [reply 3.create-model/pytorch-lab/notebooks/01-data-exploration.ipynb]: Great suggestion! Added the .format(precision=) parameter which is the newer one

File: 3.create-model/pytorch-lab/notebooks/02-model-development.ipynb (hunk 4 above)
Comment (JeetRH) [reply 3.create-model/pytorch-lab/notebooks/02-model-development.ipynb]: Made the change
